pytorch_lightning/lite/wrappers.py

```diff
@@ -50,18 +50,20 @@ def __init__(self, optimizer: Optimizer, accelerator: Accelerator) -> None:
         self.__class__ = type("Lite" + optimizer.__class__.__name__, (self.__class__, optimizer.__class__), {})
         self._optimizer = optimizer
         self._accelerator = accelerator
+        # TODO refactor to take Strategy as param, API breaking change for Lite? @
+        self._strategy = self._accelerator.training_type_plugin
```

Comment:
```suggestion
        # TODO (@awaelchli) refactor to take Strategy as param
        self._strategy = self._accelerator.training_type_plugin
```

nope, no problem. 
I anticipated this and we marked _LiteOptimizer as protected. This is not user facing so all good.
I'll handle this in a separate PR!